packages/modular-scripts/src/esbuild-scripts/start/runtime/index.ts

```diff
@@ -59,7 +59,7 @@ interface WebSocketMessage {
 }
 
 connection.onmessage = (m: MessageEvent) => {
-  const message = JSON.parse(m.data) as WebSocketMessage;
```

Comment:
Might be worth having a runtime safety check here. The simplest upgrade is to make it throw when `m.data` isn't a string - since `JSON.parse` will already throw in that situation. 

I'm not sure what the consequences are of `m.data` being non-string - those consequences should drive if this should throw or warn (or do nothing, or something else).

I say all this on the assumption that the `MessageEvent` interface is accurate - if `data` can be non-string, our code should account for that, probably (or we should known why we *don't* need to account for it).

Reply:
Thanks, I tried to improve my understanding of .data and why it's typed as any/if we need to consider it being anything but string. 

I opted to follow your suggestion to throw if not string, as JSON.parse will have done anyways. hopefully should never be the case.